variables.tf

```diff
@@ -1014,3 +1014,9 @@ variable "use_nlb" {
   default     = false
   description = "If true this will create a NLB loadbalancer instead of a classic VPC ELB"
 }
+
+variable "enable_s3_access_logging" {
```

Comment:
Please ask in the Devops slack channel if people think this should be enabled by default.